jetty-start/src/main/java/org/eclipse/jetty/start/CommandLineBuilder.java

```diff
@@ -62,31 +62,7 @@ public static String findJavaBin()
      */
     public static String quote(String arg)
     {
-        boolean needsQuoting = (arg.indexOf(' ') >= 0) || (arg.indexOf('"') >= 0);
-        if (!needsQuoting)
-        {
-            return arg;
-        }
-        StringBuilder buf = new StringBuilder();
-        // buf.append('"');
-        boolean escaped = false;
-        boolean quoted = false;
-        for (char c : arg.toCharArray())
-        {
-            if (!quoted && !escaped && ((c == '"') || (c == ' ')))
-            {
-                buf.append("\\");
-            }
-            // don't quote text in single quotes
-            if (!escaped && (c == '\''))
-            {
-                quoted = !quoted;
-            }
-            escaped = (c == '\\');
-            buf.append(c);
-        }
-        // buf.append('"');
-        return buf.toString();
+        return "'" + arg + "'";
```

Comment:
is this method used?

Reply:
Not anymore by us.
I left it in place with a naive implementation.
Guess I could flag it for deprecation too.

Reply:
I think we can just delete it.